### DISCUSSION TITLE
libimhex.so  missing from nightly flatpak builds

### DISCUSSION THREAD

**Discussion by presianbg** [WerWolv/ImHex#346] [Get Help] [answered]
Hi,

I'm not able to open a bug issue on this repo - results in `HTTP ERROR 400`, so I'm writing here.
Looks like libimhex.so is not being included in the nightly flatpak build.

I want to try the pattern files, but because of #318 I need to use the nightly build.

Cheers,
PY

**Comment by WerWolv** [comment, marked as answer]
It actually was missing from every single Linux build... Thank you!
Should be fixed now.
